include/deal.II/physics/elasticity/standard_tensors.h

```diff
@@ -201,12 +201,12 @@ namespace Physics
        *  \{ \bullet \} : \hat{\mathcal{P}}
        *    \dealcoloneq J^{-2/\textrm{dim}} \left[ \{ \bullet \} -
        * \frac{1}{\textrm{dim}}\left[\mathbf{C} : \{ \bullet \}\right]
-       * \mathbf{C}^{-1} \right] = \texttt{Dev\_P} \left( \{ \bullet \} \right)
+       * \mathbf{C}^{-1} \right] = \mathrm{Dev\_P} \left( \{ \bullet \} \right)
```

Comment:
should these be ``\mathtt`` ?

Reply:
I wrote this PR before I learned about `\mathtt`

Reply:
would you mind changing them?

Reply:
Fixed!